test/plots/rectanglePlotTests.ts

```diff
@@ -142,6 +142,17 @@ describe("Plots", () => {
         assert.strictEqual(entities[1].index, 2, "the entity of index 2 is retrieved");
         svg.remove();
       });
+
+      it("retrieves undefined from entityNearest when no entities are rendered", () => {
+        plot.addDataset(new Plottable.Dataset([]));
+        plot.renderTo(svg);
+        let closest = plot.entityNearest({
+          x: xScale.scale(1),
+          y: xScale.scale(1)
```

Comment:
`xScale` for y? Any why not just use direct pixel values like `plot.width()/2`, `plot.height()/2`?